project_timeline_hr_timesheet/static/src/scss/project_timeline_hr_timesheet.scss

```diff
@@ -9,10 +9,10 @@
 .o_project_timeline_hr_timesheet_progress .div {
```

Comment:
This would produce `.o_project_timeline_hr_timesheet_progress .o_project_timeline_hr_timesheet_progress .div {` in CSS which is not the intended behaviour I believe